packages/react-components/src/components/RTE/RTERibbonButtons.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Theme } from '@fluentui/react';
+import { KnownRibbonButtonKey, LocalizedStrings, RibbonButton, getButtons } from 'roosterjs-react';
+import { ribbonButtonStyle, ribbonDividerStyle } from '../styles/RichTextEditor.styles';
+import { RTESendBoxStrings } from './RTESendBox';
+
+const dividerRibbonButton = (theme: Theme): RibbonButton<string> => {
+  return {
+    key: 'Divider',
```

Comment:
Is there an aria label string needed for this?

Reply:
I think we can skip it as there is no aria-label for a divider in Teams and add later in case if we need to.